FirefoxLite_Downloader_Library/src/main/java/com/example/firefoxlitedownloaderlibrary/MainFunctionAccessFile.java

```diff
@@ -0,0 +1,25 @@
+package com.example.firefoxlitedownloaderlibrary;
+
+import java.util.concurrent.TimeUnit;
+
+import androidx.work.OneTimeWorkRequest;
+import androidx.work.WorkManager;
+
+public class MainFunctionAccessFile {
+
+
+    public void download()
```

Comment:
we can discuss this coding style later.  Suggest to use CheckStyle for Java, and ktlint for Kotlin

Reply:
@cnevinc Will add this in the next pull request.